db/migrate/20220429051320_add_user_ref_to_comments.rb

```diff
@@ -1,5 +1,6 @@
 class AddUserRefToComments < ActiveRecord::Migration[7.0]
   def change
-    add_reference :comments, :user, null: false, foreign_key: true
+    add_reference :comments, :user, null: false, index: true
+    add_foreign_key :comments, :users, column: :author_id, on_delete: :cascade
```

Comment:
Same here 

![image](https://user-images.githubusercontent.com/65068771/167687135-e5f36852-9263-4b1b-a100-daa55df6e380.png)


- [ ] Kindly correct this migration as is throwing an error. :point_up: 
Check this example :point_down: 

```rb 
class AddUserRefToComments < ActiveRecord::Migration[7.0]  def change
    add_reference :comments, :author, null: false, index: true, foreign_key: { to_table: 'users' }
  end
end 
```